labs/ipfs/api-cli/pin-operations.mdx

```diff
@@ -0,0 +1,213 @@
+    - To avoid loss of access to the pinned content, the URL should be **stable and persistent**.
+    - The server hosting the content must provide certain HTTP headers in its response, such as `Content-Length` (the size of the content) and `Content-Type` (the type of content, such as `png`for images or `html` for text). You can verify whether your content is missing either headers by running the `curl -I` command, followed by the URL of the content you are trying to pin.
+</Message>
+
+```
+  scw ipfs pin create-by-url volume-id=11111111-1111-1111-1111-111111111111 url=https://upload.wikimedia.org/wikipedia/commons/4/47/PNG_transparency_demonstration_1.png name=MyNewPinByURL
+```
+
+You should get a response like the following, providing details about the newly created pin.
+
+```
+  PinID           22222222-2222-2222-2222-222222222222
+  Status          pinned_ok
+  CreatedAt       now
+  Cid.Cid         -
+  Cid.Name        MyNewPinByUrl
+```
+
+Alternatively, you can also run the following API command. Just make sure to replace the example values with your own, using the API configuration table below:
+
+```
+  curl -X POST \
+    -H "X-Auth-Token: $SCW_SECRET_KEY" \
+    -H "Content-Type: application/json" \
+    -d '{
+      "name": "MyNewPinByURL",
+      "url": "https://upload.wikimedia.org/wikipedia/commons/4/47/PNG_transparency_demonstration_1.png",
+      "volume_id": "6170692e-7363-616c-6577-61792e636f6d"
+    }' \
+    "https://api.scaleway.com/ipfs/v1alpha1/regions/$REGION_ID/pins/create-by-url"
+```
+
+In both cases, a CID will be provided for your newly pinned content.
+
+## Retrieving information about a pin
+
+This operation allows you to retrieve information about a given pin. You can either retrieve the CID associated with the pin ID or discover the CID behind the URL you provided.
+
+Run the following command to retrieve information about a pin using CLI. Just make sure to replace the example values with your own:
+
+```
+  scw ipfs pin get pin-id=22222222-2222-2222-2222-222222222222 volume-id=11111111-1111-1111-1111-111111111111
+```
+
+You should get a response like the following, providing details about the pin:
+
+```
+Status          pinned
+CreatedAt       5 minutes ago
+Cid.Cid         QmaGNR5PedL8kiTUMtsPeFFodL9HnYBMuUqAEQrv6CQH4F
+Cid.Name        value:"MyNewPinByCID"
+```
+
+<Message type="note">
+  If the content is successfully pinned, the response will provide the CID of your content pointed by the URL. If it fails, no CID will be returned. You will still be able to retrieve the URL of the pinned content so you can retry.
+</Message>
+
+## Deleting a pin
+
+This operation allows you to delete a given pin within a volume using its pin ID. If you were the last owner of the associated content, the content will be permanently deleted.
```

Comment:
`If you were the last owner of the associated content` this sentence can be deleted as it explains an implementation detail that is not important.